libsrc/supervision/joy/supervision-stdjoy.s

```diff
@@ -77,5 +77,6 @@ COUNT:
 
 READJOY:
         lda     sv_control
+	eor     #$FF
```

Comment:
Sorry for the nitpicking but you need to use spaces instead of a tab char for indentation to have me merge this pr as-is.